ppb/scenes.py

```diff
@@ -20,33 +27,65 @@ def __init__(self, engine, **kwargs):
         }
 
     def render(self):
+        """
+        Renders all the objects in the groups
+
+        :return: Iterable representing all the drawn rectangles
+        """
         window = self.engine.display
         rects = [g.draw(window, self.background) for g in self.groups.values()]
         return chain(*rects)
 
     def handle_event(self, event):
+        """
+        Calls the specified callback function from the callback map
+
+        :param event: The event that is supposed to be handled
+        """
         self.callback_map.get(event.type, self.__null__)(event)
 
     def publish_event(self, event):
         pass
 
     def simulate(self, time_delta: float):
+        """
+        Update all objects in the group calling their update function
+
+        :param time_delta: Time that has passed since the last update
+        """
         for group in self.groups.values():
             group.update(time_delta)
 
     def change(self):
         """
         Default case, override in subclass as necessary.
+
+        :return: Tuple specifying if the scene is running and what the scene class is
         """
         return self.running, {"scene_class": self.next}
 
     def __null__(self, event):
+        """
+        Does nothing
+
+        :param event: unused
+        """
         pass
 
     def __quit__(self, event):
+        """
+        Quits the current scene
```

Comment:
This doesn't actually quit the scene, it just sets it's running and quit values appropriately.